subsys/bluetooth/host/hci_core.c

```diff
@@ -3370,6 +3377,40 @@ static void le_adv_report(struct net_buf *buf)
 }
 #endif /* CONFIG_BT_OBSERVER */
 
+#if defined(CONFIG_BT_HCI_VENDOR_EVENT_CALLBACK)
+int bt_hci_register_vnd_evt_cb(bt_hci_vnd_evt_cb_t cb)
+{
+	hci_vnd_evt_cb = cb;
+	return 0;
+}
+#endif /* CONFIG_BT_HCI_VENDOR_EVENT_CALLBACK */
+
+static inline void default_hci_vendor_event(struct net_buf *buf)
+{
+	/* do nothing at present time */
```

Comment:
I guess it wouldn't hurt to add a BT_DBG() here about the unhandled vendor event.

Reply:
I will add that

Reply:
BT_DBG or BT_WARN? We use BT_WARN handle_events?

Reply:
BT_WARN sounds appropriate